telemetry/data_window.go

```diff
@@ -253,7 +254,8 @@ func RotateSubWindow() {
 
 	err := readSQLMetric(time.Now(), &thisSubWindow.SQLUsage)
 	if err != nil {
-		logutil.BgLogger().Info("Error exists when getting the SQL Metric.")
+		logutil.BgLogger().Info("Error exists when getting the SQL Metric.",
+			zap.String("error", err.Error()))
```

Comment:
How about just using `zap.Error(err)`?

Reply:
Changed this on a few other places as well in f4a975dc48fe3281199a122242a5ec4418194598. Looks more consistent now.

```
$ jq 'select(.error) | select(.error | contains("Prometheus"))' tidb.log
```
```json
{
  "level": "INFO",
  "time": "2023/01/27 10:00:35.554 +01:00",
  "caller": "data_slow_query.go:67",
  "message": "Failed to get Slow Query Stats",
  "error": "[domain:9009]Prometheus address is not set in PD and etcd"
}
{
  "level": "INFO",
  "time": "2023/01/27 10:01:36.223 +01:00",
  "caller": "data_window.go:257",
  "message": "Error exists when getting the SQL Metric.",
  "error": "[domain:9009]Prometheus address is not set in PD and etcd"
}
{
  "level": "INFO",
  "time": "2023/01/27 10:02:36.225 +01:00",
  "caller": "data_window.go:257",
  "message": "Error exists when getting the SQL Metric.",
  "error": "[domain:9009]Prometheus address is not set in PD and etcd"
}
```

Reply:
PTAL @breezewish @hawkingrei